moped-editor/src/views/projects/projectView/ProjectNameForm.js

```diff
@@ -1,195 +1,184 @@
-            value={secondaryName || ""}
-            placeholder={"Secondary name"}
-            multiline={false}
-            rows={1}
-            onChange={handleSecondaryNameChange}
-            InputLabelProps={{
-              shrink: true,
-            }}
-            InputProps={{
-              classes: {
-                input: classes.projectSecondaryNameEditField,
-              },
-            }}
-          />
-        </form>
-      </Grid>
-
-      {/* Accept / Cancel icons.
-      This grid item gets a minimum width to prevent it from reflowing onto two lines. */}
-      <Grid
-        item
-        container
-        direction="row"
-        justifyContent="flex-end"
-        alignItems="flex-end"
-        xs={12}
-        sm={1}
-        sx={{ minWidth: "77px" }}
+    <>
+      <Box
+        component="form"
+        sx={{ minWidth: "100%" }}
+        onSubmit={handleSubmit(handleSave)}
       >
-        <Icon
-          className={classes.editIcons}
-          onClick={(e) => handleAcceptClick(e)}
-        >
-          check
-        </Icon>
-        <Icon
-          className={classes.editIcons}
-          onClick={(e) => handleCancelClick(e)}
-        >
-          close
-        </Icon>
-      </Grid>
-
-      {/* The status badge. Here, we're going to jog it down a bit to make it visually centered
-      along the horizontal midline of the project name input field. */}
-      <Grid item xs={12} md={2} container alignItems="flex-end">
-        <ProjectStatusBadge
-          phaseKey={props.currentPhase?.phase_key}
-          phaseName={props.currentPhase?.phase_name}
-        />
-      </Grid>
-    </Grid>
+        <Grid container spacing={2}>
+          <Grid item xs={12} sm={5}>
+            <ControlledTextInput
+              autoFocus
+              variant="standard"
+              fullWidth
+              id="project_name"
+              label="Project name"
+              name="projectName"
+              placeholder="Enter project name"
+              control={control}
+              error={!!errors?.projectName}
+              helperText={errors?.projectName?.message}
+              InputLabelProps={{
+                shrink: true,
+              }}
+              InputProps={{
+                sx: inputStyles,
+              }}
+            />
+          </Grid>
+          <Grid item xs={12} sm={3}>
+            <ControlledTextInput
+              variant="standard"
+              fullWidth
+              autoFocus
+              id="secondary_name"
+              name="projectSecondaryName"
+              placeholder="Secondary name"
+              label="Secondary name"
+              control={control}
+              error={!!errors?.projectSecondaryName}
+              helperText={errors?.projectSecondaryName?.message}
+              InputLabelProps={{
+                shrink: true,
+              }}
+              InputProps={{
+                sx: inputStyles,
+              }}
+            />
```

Comment:
it would be nice to disable these inputs while the form is submitting

Reply:
🙏 this is a nice detail so I updated in https://github.com/cityofaustin/atd-moped/pull/1557